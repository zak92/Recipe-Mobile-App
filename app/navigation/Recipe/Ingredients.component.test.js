import React from 'react';
import { render } from '@testing-library/react-native'; // json rendering
import Ingredients from './Ingredients.component';


describe('Ingredients', () => {
  it("should match snapshot", () => {
    const snapIngredients = render(<Ingredients />).toJSON(); // render this file to JSON
    expect(snapIngredients).toMatchSnapshot(); // this snapshot should match the previous snapshot

  });
  it('displays the passed-in ingredient', () => {
    const { queryByText } = render(
      <Ingredients ingredients="2 heaps of chocolate ice-cream" />
    );
    expect(queryByText('2 heaps of chocolate ice-cream')).not.toBeNull();
  });

});