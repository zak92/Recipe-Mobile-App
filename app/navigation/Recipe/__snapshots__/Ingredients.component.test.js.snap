// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ingredients should match snapshot 1`] = `
<View
  style={
    Object {
      "borderBottomColor": "lightgrey",
      "borderBottomWidth": 1,
      "paddingBottom": "2%",
      "paddingLeft": "5%",
      "paddingRight": "5%",
      "paddingTop": "2%",
    }
  }
>
  <Text
    style={
      Object {
        "color": "black",
        "fontFamily": "Roboto_400Regular",
        "fontSize": 16,
      }
    }
    testID="ingredients"
  />
</View>
`;
